cmd/legacy_main.go

```diff
@@ -113,7 +113,11 @@ func getConfigForUserAgent(mountConfig *cfg.Config) string {
 	if cfg.IsParallelDownloadsEnabled(mountConfig) {
 		isParallelDownloadsEnabled = "1"
 	}
-	return fmt.Sprintf("%s:%s:%s", isFileCacheEnabled, isFileCacheForRangeReadEnabled, isParallelDownloadsEnabled)
+	areStreamingWritesEnabled := "0"
+	if cfg.AreStreamingWritesEnabled(mountConfig) {
```

Comment:
can we check the config directly here instead of seperate method
c.Write.EnableStreamingWrites?
why to write a seperate method for checking one condition.